addons/mass_mailing/views/res_config_settings_views.xml

```diff
@@ -53,6 +53,17 @@
                                     </div>
                                 </div>
                             </div>
+                            <div class="col-md-6 o_setting_box col-xs-12" name="no_mass_mailing_reports_setting_container">
```

Comment:
Generally we do not change view / template in stable... I do not think this change will cause issue (as well as `report_unsubscribed`), but keep this comment not solved so smarter people think about that during the review :)